docker-compose.yml

```diff
@@ -10,20 +10,20 @@ services:
         condition: service_healthy
     env_file: ./docker.env
     ports:
-      - "5000:5000"
+      - 5000:5000
     restart: unless-stopped
   rabbitmq:
     image: rabbitmq:3.8-management-alpine
-    container_name: 'rabbitmq'
+    container_name: rabbitmq
     ports:
       - 5672:5672
       - 15672:15672
     # A stable hostname is important to prevent the volume from continually growing on
     # every restart. See issue described here: https://stackoverflow.com/questions/63115192/stale-rabbitmq-data-queue-files-eating-up-disk
-    hostname: 'rabbitmq'
+    hostname: rabbitmq
     volumes:
-      - rabbitmq-data/:/var/lib/rabbitmq/
-      - rabbitmq-logs/:/var/log/rabbitmq
+      - rabbitmq-data:/var/lib/rabbitmq/
```

Comment:
This is the most essential part of the fix, to play nice with more recent versions of docker-compose.